client/src/components/existing-request/existing-request-search.vue

```diff
@@ -26,10 +28,11 @@
                       validate-on-blur />
       </v-col>
     </v-row>
-    <v-row align="center" class="ml-n9 mt-2" dense>
```

Comment:
Got rid of negative margin (that didn't work when screen was resized anyway), yay!